app/Http/Kernel.php

```diff
@@ -31,27 +31,29 @@ class Kernel extends HttpKernel
         'auth.basic' => \Illuminate\Auth\Middleware\AuthenticateWithBasicAuth::class,
         'guest' => \App\Http\Middleware\RedirectIfAuthenticated::class,
         'member' => \App\Http\Middleware\RedirectIfNotAMember::class,
+        'participant' => \App\Http\Middleware\RedirectIfNotAParticipant::class,
         'cors' => \App\Http\Middleware\CORS::class,
         'bindings' => \Illuminate\Routing\Middleware\SubstituteBindings::class,
         'can' => \Illuminate\Auth\Middleware\Authorize::class,
         'privacy' => \App\Http\Middleware\CheckPrivacy::class,
+        'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
     ];
 
     protected $middlewareGroups = [
         'web' => [
-
             \App\Http\Middleware\EncryptCookies::class,
             \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
             \Illuminate\Session\Middleware\StartSession::class,
             \Illuminate\View\Middleware\ShareErrorsFromSession::class,
             \Illuminate\Routing\Middleware\SubstituteBindings::class,
             \App\Http\Middleware\CheckPrivacy::class,
-            // \App\Http\Middleware\VerifyCsrfToken::class,
+            \App\Http\Middleware\VerifyCsrfToken::class,
```

Comment:
Maybe use the string names here as well, or the FQN on the api side. Now web uses FQN while api uses string names.